diamond-api/src/bin/diamond-api.rs

```diff
@@ -44,7 +44,7 @@ fn run(args: Args) -> io::Result<()> {
             .default_service(web::route().to(HttpResponse::NotFound))
     })
     .bind(listen)?
-    .run()?;
+    .run();
```

Comment:
Does it work? AFAIK `run()` just returns a future and quits.

Reply:
👍

Reply:
fixed with creating a system before it.